types/navermaps/index.d.ts

```diff
@@ -133,11 +133,11 @@ declare namespace naver.maps {
         zoomOrigin?: any;
     }
     interface MarkerOptions {
-        animation?: any;
-        map?: Map | undefined;
-        position?: any;
-        icon?: any;
-        shape?: any;
+        animation?: Animation;
+        map: Map | undefined;
+        position: any;
```

Comment:
This type can probably be updated as well:

```suggestion
        position: Coord | CoordLiteral;
```

Reply:
My mistake, thank you for reviewing! 😊